strawberryfields/apps/train/__init__.py

```diff
@@ -3,19 +3,87 @@
 
 .. currentmodule:: strawberryfields.apps.train
 
+A GBS device can be treated as a variational quantum device with trainable squeezing,
```

Comment:
The easiest way to visualise the changes is by going here:
https://strawberryfields--414.org.readthedocs.build/en/414/code/api/strawberryfields.apps.train.html#module-strawberryfields.apps.train

Reply:
This is so helpful Tom! :mage: